test/ambuda/views/test_auth.py

```diff
@@ -140,8 +140,9 @@ def test_reset_password_from_token(client):
 
     user_id = user.id
     raw_token = auth._create_reset_token(user_id)
+    bad_token = "bad-token"
 
-    r = client.get(f"/reset-password/akprasad/bad-token")
+    r = client.get(f"/reset-password/akprasad/{bad_token}")
```

Comment:
Let's use what's on `main` without the `f"` prefix

Reply:
done